1_fetch/src/get_sb_data.R

```diff
@@ -186,6 +193,7 @@ prep_feature_vars <- function(sb_var_data, sites, retain_vars) {
     subset(!(COMID %in% comid_out_conus)) %>%
     group_by(COMID, unit, year, new_class) %>%
     mutate(lc_adj = (value / lc_sum) * 100) %>%
+    ungroup() %>%
     select(COMID, unit, new_class, lc_adj) %>%
     group_by(COMID, unit, new_class) %>%
     summarise(value = mean(lc_adj), .groups = "drop") %>%
```

Comment:
Adding comment on the nearest line that I can comment on.

Can you add `by = COMID` to these joins to remove the message that it's joining by COMID?
    left_join(land_cover) %>%
    left_join(dams) %>%
    left_join(weather) %>%
    left_join(wildfire)